src/server_manager/scripts/get_version.mjs

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import url from "url";
+import path from "path";
+import {readFile} from "fs/promises";
+
+const __filename = url.fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+export async function get_version() {
+  const {version} = JSON.parse(await readFile(path.resolve(__dirname, "../package.json")));
+
+  return version;
+}
+
+async function main() {
+  console.log(await get_version());
+}
+
+if (import.meta.url === url.pathToFileURL(process.argv[1]).href) {
+  (async function() {
```

Comment:
Can we just call `main()` here?

Reply:
`return` ensures the promise returned from `main` gets resolved in the process

Reply:
But when I try `main()` directly, the promise is still resolved if you call it in CLI. Cuz from the caller's perspective, `(async ...)()` and `main()` seems to be the same because main can also be treated as an `(async...)()` call.

Reply:
It's basically a poor man's `top-level await`: https://v8.dev/features/top-level-await